sympy/solvers/solveset.py

```diff
@@ -855,6 +863,9 @@ def solveset(f, symbol=None):
 
     f = sympify(f)
 
+    if f is S.false:
+        return EmptySet()
```

Comment:
Currently:
```
>>> x = Symbol('x', positive=True)
>>> solveset(Eq(x, -1), x)
False
```
But Is not `solveset` supposed to return always a `set` ? That's why i did this.